HW0/P4/P4.py

```diff
@@ -29,46 +29,94 @@
     #
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
+
+    s_true = np.loadtxt("P4_trajectory.txt", delimiter = ',')
+
+    x_coords, y_coords, z_coords = s_true[:,0], s_true[:,1], s_true[:, 2]
+
     #####################
```

Comment:
Naming the coords seem not necessary here I think as you are only using this coords once